annotated-projects/PXD000923/PXD000923.sdrf.tsv

```diff
@@ -0,0 +1,11 @@
+Sample 5	Oryza sativa	Flower	not applicable	not applicable	1	run 9	GZ63S	PXD000923	LHY_T5.wiff	http://ftp.pride.ebi.ac.uk/pride/data/archive/2014/07/PXD000923/raw_data.zip	5	1	NT=label free sample	AC=MS:1000932;NT=TripleTOF 5600	AC=MS:1001251;NT=Trypsin	NT=Carbamidomethyl;AC=UNIMOD:4;TA=C;MT=Fixed	NT=Oxidation;MT=Variable;TA=M;AC=UNIMOD:35	NT=Phospho;AC=UNIMOD:21;TA=S;MT=Variable	NT=Phospho;AC=UNIMOD:21;TA=T;MT=Variable	NT=Phospho;AC=UNIMOD:21;TA=Y;MT=Variable	15 ppm	0.05 Da
+Sample 6	Oryza sativa	Flower	not applicable	not applicable	2	run 11	GZ63S	PXD000923	LHY_T6.wiff	http://ftp.pride.ebi.ac.uk/pride/data/archive/2014/07/PXD000923/raw_data.zip	6	1	NT=label free sample	AC=MS:1000932;NT=TripleTOF 5600	AC=MS:1001251;NT=Trypsin	NT=Carbamidomethyl;AC=UNIMOD:4;TA=C;MT=Fixed	NT=Oxidation;MT=Variable;TA=M;AC=UNIMOD:35	NT=Phospho;AC=UNIMOD:21;TA=S;MT=Variable	NT=Phospho;AC=UNIMOD:21;TA=T;MT=Variable	NT=Phospho;AC=UNIMOD:21;TA=Y;MT=Variable	15 ppm	0.05 Da
+Sample 7	Oryza sativa	Flower	not applicable	not applicable	2	run 13	GZ63S	PXD000923	LHY_T7.wiff	http://ftp.pride.ebi.ac.uk/pride/data/archive/2014/07/PXD000923/raw_data.zip	7	1	NT=label free sample	AC=MS:1000932;NT=TripleTOF 5600	AC=MS:1001251;NT=Trypsin	NT=Carbamidomethyl;AC=UNIMOD:4;TA=C;MT=Fixed	NT=Oxidation;MT=Variable;TA=M;AC=UNIMOD:35	NT=Phospho;AC=UNIMOD:21;TA=S;MT=Variable	NT=Phospho;AC=UNIMOD:21;TA=T;MT=Variable	NT=Phospho;AC=UNIMOD:21;TA=Y;MT=Variable	15 ppm	0.05 Da
+Sample 8	Oryza sativa	Flower	not applicable	not applicable	2	run 15	GZ63S	PXD000923	LHY_T8.wiff	http://ftp.pride.ebi.ac.uk/pride/data/archive/2014/07/PXD000923/raw_data.zip	8	1	NT=label free sample	AC=MS:1000932;NT=TripleTOF 5600	AC=MS:1001251;NT=Trypsin	NT=Carbamidomethyl;AC=UNIMOD:4;TA=C;MT=Fixed	NT=Oxidation;MT=Variable;TA=M;AC=UNIMOD:35	NT=Phospho;AC=UNIMOD:21;TA=S;MT=Variable	NT=Phospho;AC=UNIMOD:21;TA=T;MT=Variable	NT=Phospho;AC=UNIMOD:21;TA=Y;MT=Variable	15 ppm	0.05 Da
+Sample 9	Oryza sativa	Flower	not applicable	not applicable	2	run 17	GZ63S	PXD000923	LHY_T9.wiff	http://ftp.pride.ebi.ac.uk/pride/data/archive/2014/07/PXD000923/raw_data.zip	9	1	NT=label free sample	AC=MS:1000932;NT=TripleTOF 5600	AC=MS:1001251;NT=Trypsin	NT=Carbamidomethyl;AC=UNIMOD:4;TA=C;MT=Fixed	NT=Oxidation;MT=Variable;TA=M;AC=UNIMOD:35	NT=Phospho;AC=UNIMOD:21;TA=S;MT=Variable	NT=Phospho;AC=UNIMOD:21;TA=T;MT=Variable	NT=Phospho;AC=UNIMOD:21;TA=Y;MT=Variable	15 ppm	0.05 Da
+Sample 10	Oryza sativa	Flower	not applicable	not applicable	2	run 19	GZ63S	PXD000923	LHY_T10.wiff	http://ftp.pride.ebi.ac.uk/pride/data/archive/2014/07/PXD000923/raw_data.zip	10	1	NT=label free sample	AC=MS:1000932;NT=TripleTOF 5600	AC=MS:1001251;NT=Trypsin	NT=Carbamidomethyl;AC=UNIMOD:4;TA=C;MT=Fixed	NT=Oxidation;MT=Variable;TA=M;AC=UNIMOD:35	NT=Phospho;AC=UNIMOD:21;TA=S;MT=Variable	NT=Phospho;AC=UNIMOD:21;TA=T;MT=Variable	NT=Phospho;AC=UNIMOD:21;TA=Y;MT=Variable	15 ppm	0.05 Da
```

Comment:
It seems that the first 5 files (run 1 - run 9 or run 11 - run 19)are from the same fraction group under a sample. So their source name should be the same.   Other annotations look good.

Reply:
@daichengxin Please have a look into the datasets, At least 9 out of 10 looks ok, so if possible please merge it with the master. Thank you